src/components/views/ClassRoster/index.jsx

```diff
@@ -2,13 +2,13 @@ import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { logError } from '@edx/frontend-platform/logging';
-import DashboardLaunchButton from '../DashboardLaunchButton';
-import TableFilter from '../TableFilter';
+import DashboardLaunchButton from '../../shared/DashboardLaunchButton';
+import TableFilter from '../../shared/TableFilter';
+import Table from '../../shared/Table';
+import { columns } from './columns';
+import { defaultErrorMessage } from '../../../constants';
```

Comment:
I suggest to add a space here.
```suggestion

import DashboardLaunchButton from '../../shared/DashboardLaunchButton';
import TableFilter from '../../shared/TableFilter';
import Table from '../../shared/Table';
import { columns } from './columns';
import { defaultErrorMessage } from '../../../constants';
```